src/components/Footer/Channels/Channels.style.tsx

```diff
@@ -15,3 +15,7 @@ export const ChannelButtonsWrap = styled.div<WrapProps>`
   height: ${(props: WrapProps) => (props.isFooter ? '30px' : '96px')};
   margin-top: 15px;
 `;
+
+export const ChannelButton = styled.img`
```

Comment:
요거 `next/image` 로 사용 안하는 이유 있나요??!

Reply:
그 이유는 .. 제가 몰랐기 때문입니다 ...

Reply:
@SeojinSeojin 
방법은 너무 좋습니다 !! 저도 동적으로 svg 파일을 조작하는 코드가 아니라면, Image 태그를 이용해서 스타일링 할게요!

---

(완전 가벼운 제안)
혹시 포인터 커서를 위한 SC인데, 이름에 "포인터 커서야" 를 알려주는 형용사를 추가하는 건 어떨까요?

Reply:
넥제의 이미지 컴포넌트는 레이지 로딩 관련한 쓰임새가 큰 것으로 보이는데, 아이콘은 크기가 작아서 레이지 로딩이 엄청 필수적이지는 않을듯혀..!!
주햄 말해준 대로 아이콘은 Image 태그 써봐도 좋을 것 같어

Reply:
네이밍 : Chat GPT에게 함 물어봤어 .. 

Sure, here are some possible naming conventions for styled components that include the cursor: pointer property:

1. Clickable: This name indicates that the component is designed to be clicked, and is a simple and clear way to describe its functionality.
2. Button: If the component is a button or has a button-like appearance, this name can be appropriate. It implies interactivity and can be easily recognized by users.
Link: Similar to the Button naming convention, Link implies that the component is interactive and clickable. This name can be used for components that resemble hyperlinks or anchor tags.
ClickableIcon: If the component is an icon or has an icon-like appearance, this name can be appropriate. The Clickable prefix indicates that the icon is interactive, while the Icon suffix indicates its appearance.
Interactive: This name is a more general term that can be used for any component that is interactive, including those that use the cursor: pointer property. However, it may not be as descriptive as the other naming conventions listed above.
It's important to note that the naming conventions used for styled components can vary depending on the project and the team's preferences. Ultimately, the most important thing is to choose names that are clear, concise, and accurately describe the component's functionality

Reply:
넥제의 이미지 컴포넌트가 내가 알기로, 레이지 로딩 이외에도
- 리사이징 및 이미지 압축으로 이미지 최적화
- 레이아웃 지정
- 이미지 캐싱

등 다양하게 기능을 제공해주는 걸로 알고 있어!

그래서 넥제에서는 img 태그보다는 next/image 모듈을 사용하는 게 좋다고 알고있어서 이야기 해봤어- :))
(사실 너 말대루 크기도 작고 사이즈도 작아서 "굳이" 이기는 하지만~~.~)

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ네이밍 `ClickableChannelButton` 좋네 ^^~:ㅣ:$

Reply:
아 ~~~ 버튼이 이미 그런 의미를 담고있는거구나!

Reply:
최적화와 캐싱도 지원해주는군 ..!! 
스타일을 줄 때 스타일드컴포넌트랑 같이 못 쓰는게 아쉬운데 .. 

<스타일을 가진 div>
  <넥제 Image 태그 />
</스타일을 가진 div>

이런 식의 컴포넌트를 만들고 ClickableImage라는 이름을 붙여서 쓰는 걸로 .. 해보면 어떨까 ..?!

Reply:
헐 그렇게 쓸 수 있구나!! 

너무나 쉽고 편하고 멋지네 ...

당장 내일 아침에 글케 고쳐서 푸쉬할게유

늦은 시간까지 봐줘서 넘넘 고마우이!!!!

Reply:
직접 해보니 잘 된다 !! 
(+ next/image는 width, height 프로퍼티 혹은 layout="fill" 속성이 필수래서, `width="30" height="30"` 을 명시해줘야 될 것 같으네?)

---

사담이지만, 커비가 Cloudflare Pages 이야기 해줄 떄, SSR을 지원 안 해준다고 했잖아?
나는 next/image 지원이 안 되는 게 조금 치명적일 수 있다고 생각했어 :(..
공식 홈페이지는 솝트의 얼굴이기에... 최대한 렌더링 속도를 줄이고 여러 방면에서 최적화 해야된다고 생각하는데(이 경험 여유 될 때 꼭 가져가고파),  next/image를 못쓰는 게 많이 아쉽더라고,,!

Reply:
방금 width와 height 주었습니다!! 넘 감사합니다

그리고 next/image를 향한 마음도 잘 알았닷!! 앞으로 이미지 쓸 때 유의할게유~!